tests/components/august/test_camera.py

```diff
@@ -35,3 +39,37 @@ async def test_create_doorbell(
         assert resp.status == HTTPStatus.OK
         body = await resp.text()
         assert body == "image"
+
+
+async def test_doorbell_refresh_content_token(
+    hass: HomeAssistant, hass_client_no_auth: ClientSessionGenerator
+) -> None:
+    """Test camera image content token expired."""
+    doorbell_two = await _mock_doorbell_from_fixture(hass, "get_doorbell.json")
+
+    with patch.object(
+        doorbell_two,
+        "async_get_doorbell_image",
+        create=False,
+        side_effect=ContentTokenExpired,
+    ):
+
+        def _doorbell_return_token_expired():
+            if doorbell_two._data["contentToken"] is None:
+                raise ContentTokenExpired
+
```

Comment:
I think we want to make the first call fail and the second call successful so you know it got a new token and the test asserts we are back in a good state

Reply:
yeah i was considering that, but didn't really know how to go about it. just keep a flag/call count? 
i can give it a shot

Reply:
oh found ju can just have a list of side-effects. i also had doubly mocked it, it seems. 
pushed a change now, and can verify by logs that it first gets expired then works, but not sure if i can have it assert that middle state somehow, any idea?

Reply:
I think what you have is fine to verify the success path. Its a bit hard timing wise to assert in the middle state so I think adding another test that unconditionally patches `async_get_doorbell_image ` to have the side_effect of `ContentTokenExpired` will ensure the current test keeps working and is asserting what that it can recover.

Reply:
makes sense, i can add another test a little later